docs/topics/gradle.md

```diff
@@ -511,23 +511,23 @@ If you use [kapt](kapt.md), note that the kapt annotation processing tasks are n
 >
 {type="note"}
 
-The Kotlin plugin supports [Gradle configuration cache](https://docs.gradle.org/current/userguide/configuration_cache.html),
+The Kotlin plugin uses the [Gradle configuration cache](https://docs.gradle.org/current/userguide/configuration_cache.html),
 which speeds up the build process by reusing the results of the configuration phase.
 
-See the [Gradle documentation](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:usage)
-to learn how to enable the configuration cache. Once you enable the configuration cache feature, the Kotlin Gradle plugin will
+Consult the [Gradle documentation](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:usage)
```

Comment:
Not sure about the change from 'See' to 'Consult'. It's more formal; ans 'For more information, see' is a standart wording for such cases